code_samples/recent_activity/src/Command/MonitorRecentContentCreationCommand.php

```diff
@@ -0,0 +1,86 @@
+use Ibexa\Contracts\Core\Repository\PermissionResolver;
+use Ibexa\Contracts\Core\Repository\UserService;
+use Ibexa\Contracts\Core\Repository\Values\Content\Content;
+use Symfony\Component\Console\Command\Command;
+use Symfony\Component\Console\Input\InputInterface;
+use Symfony\Component\Console\Input\InputOption;
+use Symfony\Component\Console\Output\OutputInterface;
+use Symfony\Component\Console\Style\SymfonyStyle;
+
+class MonitorRecentContentCreationCommand extends Command
+{
+    protected static $defaultName = 'app:monitor-content-creation';
+
+    private ActivityLogServiceInterface $activityLogService;
+
+    private PermissionResolver $permissionResolver;
+
+    private UserService $userService;
+
+    public function __construct(ActivityLogServiceInterface $activityLogService, PermissionResolver $permissionResolver, UserService $userService)
+    {
+        parent::__construct(self::$defaultName);
+        $this->permissionResolver = $permissionResolver;
+        $this->userService = $userService;
+        $this->activityLogService = $activityLogService;
+    }
+
+    protected function configure(): void
+    {
+        $this->setDescription('List last 10 log entry groups with creations in the last hour')
+            ->addOption('filter', 'f', InputOption::VALUE_NONE, 'If only creation log entries should be shown in log groups');
+    }
+
+    protected function execute(InputInterface $input, OutputInterface $output): int
+    {
+        $query = new Query([
+            new Criterion\ObjectCriterion(Content::class),
+            new Criterion\ActionCriterion([ActivityLogServiceInterface::ACTION_CREATE]),
+            new Criterion\LoggedAtCriterion(new \DateTime('- 1 hour'), Criterion\LoggedAtCriterion::GTE),
+        ], [new LoggedAtSortClause(LoggedAtSortClause::DESC)], 0, 10);
+
+        $filterLogs = $input->getOption('filter');
+
+        $io = new SymfonyStyle($input, $output);
+
+        $this->permissionResolver->setCurrentUserReference($this->userService->loadUserByLogin('admin'));
+
+        foreach ($this->activityLogService->findGroups($query) as $activityLogGroup) {
+            if ($activityLogGroup->getSource()) {
+                $io->section($activityLogGroup->getSource()->getName());
+            }
+            if ($activityLogGroup->getDescription()) {
+                $io->text($activityLogGroup->getDescription());
+            }
+            $table = [];
+            foreach ($activityLogGroup->getActivityLogs() as $activityLog) {
+                if (!$filterLogs || $activityLog->getAction() === ActivityLogServiceInterface::ACTION_CREATE) {
+                    $table[] = [
+                        $activityLogGroup->getLoggedAt()->format(\DateTime::ATOM),
+                        $activityLog->getObjectId(),
+                        $activityLog->getObjectName(),
+                        $activityLog->getAction(),
+                        $activityLogGroup->getUser()->login,
+                    ];
```

Comment:
Could compare actual name with name when logged (like in code_samples/recent_activity/templates/themes/admin/activity_log/ui/my_feature/simulate.html.twig)

```suggestion
                    /** @var Content $content */
                    $content = $activityLog->getRelatedObject();
                    $name = $content && $content->getName() && $content->getName() !== $activityLog->getObjectName() ? "“{$content->getName()}” (formerly “{$activityLog->getObjectName()}”)" : "“{$activityLog->getObjectName()}”";
                    $table[] = [
                        $activityLogGroup->getLoggedAt()->format(\DateTime::ATOM),
                        $activityLog->getObjectId(),
                        $name,
                        $activityLog->getAction(),
                        $activityLogGroup->getUser()->login,
                    ];
```